api/src/test/java/org/openmrs/validator/PersonValidatorTest.java

```diff
@@ -231,5 +234,28 @@ public void validate_shouldNotFailWhenDeathCauseNotCodedIsSet() {
 
 		Assert.assertFalse(errors.hasErrors());
 	}
+	
+	@Test()
+	public void validate_shouldNotFailWhenOneAddressIsNull() {
+		Person person = new Patient(1);
+		Set<PersonAddress> addresses =  new HashSet<>();
+		addresses.add(null);
+		
+		person.setDead(true);
+		person.setCauseOfDeathNonCoded("Some text describing Cause of Death");
+		person.setPersonVoided(true);
+		person.setPersonVoidReason("voidReason");
+		person.setGender("g");
+		person.setAddresses(addresses);
+		
+		Errors errors = new BindException(person, "patient");
+		PersonValidator personValidator = new PersonValidator();
+		
+		try {
+			personValidator.validate(person, errors);
+		}catch(NullPointerException np) {
```

Comment:
Are you trying to test that a null pointer exception should be thrown when there is a null in the address collection?